code/modules/mob/living/carbon/human/species/unathi.dm

```diff
@@ -15,7 +15,7 @@
 	else, frequently even their own lives. They prefer warmer temperatures than most species and \
 	their native tongue is a heavy hissing laungage called Sinta'Unathi."
 
-	species_traits = list(LIPS)
+	species_traits = list(LIPS, NO_HAIR)
```

Comment:
They do, though.

Reply:
...Huh, so they do.

Although, while apparently the horns and hair are separated, the hair does still contains stuff like frills and secondary horns, which I think should still be immune to falling out.
I'd say it's better to make Unathi immune to hairloss entirely than having their frills start falling out in clumps.